src/applications/personalization/profile-2/tests/e2e/connectedApps.cypress.spec.js

```diff
@@ -0,0 +1,106 @@
+import { PROFILE_PATHS } from '../../constants';
```

Comment:
nit: can you rename this file to `profile.connected-apps.cypress.spec.js` to match the convention used by the other Cypress tests?